### ISSUE TITLE
Error when Topgrade update Visual Studio Code

### ISSUE THREAD

**Issue by Nieto2018** [r-darwish/topgrade#661]
**What did you expect to happen?**
...
Topgrade should update Visual Studio Code.  I tried to install it with sudo snap install --classic code and sudo apt install code

**What actually happened?**
...
Topgrade fails when try update Visual Studio Code

**Additional details:**
- Which operating system or Linux distribution are you using? Zorin 15
- How did you install Topgrade?
1. sudo apt-get install cargo
2. sudo cargo install topgrade

Run `topgrade --dry-run` to see which commands Topgrade is running. If the command seems wrong and you know why pleaes tell us so.
If the command seems fine try to run it yourself and tell us if you got a different result from Topgrade.

<details><pre>

2021-02-24T11:41:26.996Z DEBUG topgrade::config > Configuration at /home/nieto/.config/topgrade.toml
 2021-02-24T11:41:26.996Z DEBUG topgrade::config > Loaded configuration: ConfigFile { pre_commands: None, post_commands: None, commands: None, git_repos: None, predefined_git_repos: None, disable: None, ignore_failures: None, remote_topgrades: None, remote_topgrade_path: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, assume_yes: None, yay_arguments: None, no_retry: None, run_in_tmux: None, cleanup: None, notify_each_step: None, accept_all_windows_updates: None, bashit_branch: None, only: None, composer: None, brew: None, linux: None, git: None, windows: None, vagrant: None }
 2021-02-24T11:41:26.996Z DEBUG topgrade::utils  > Detected "/usr/bin/notify-send" as "notify-send"
 2021-02-24T11:41:26.996Z DEBUG topgrade         > Version: 6.6.0
 2021-02-24T11:41:26.996Z DEBUG topgrade         > OS: x86_64-unknown-linux-gnu
 2021-02-24T11:41:26.996Z DEBUG topgrade         > Args { inner: ["topgrade", "-v"] }
 2021-02-24T11:41:26.996Z DEBUG topgrade         > Binary path: Ok("/home/nieto/.cargo/bin/topgrade")
 2021-02-24T11:41:26.996Z DEBUG topgrade         > Self Update: false
 2021-02-24T11:41:26.996Z DEBUG topgrade::utils  > Detected "/usr/bin/git" as "git"
 2021-02-24T11:41:26.996Z DEBUG topgrade::utils  > Cannot find "doas"
 2021-02-24T11:41:26.996Z DEBUG topgrade::utils  > Detected "/usr/bin/sudo" as "sudo"
 2021-02-24T11:41:26.996Z DEBUG topgrade::utils  > Cannot find "pwsh"
 2021-02-24T11:41:26.996Z DEBUG topgrade::utils  > Cannot find "powershell"
 2021-02-24T11:41:26.996Z DEBUG topgrade::runner > Step "System update"

―― 12:41:26 - System update ――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-02-24T11:41:26.996Z DEBUG topgrade::utils  > Cannot find "apt-fast"
 2021-02-24T11:41:26.996Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "/usr/bin/apt-get" "update"
Obj:1 http://repo.mysql.com/apt/debian stretch InRelease
Obj:2 http://ppa.launchpad.net/cwchien/gradle/ubuntu bionic InRelease                                                                                                                                                                        
Obj:3 https://download.docker.com/linux/ubuntu bionic InRelease                                                                                                                                                                              
Obj:4 http://packages.microsoft.com/repos/edge stable InRelease                                                                                                                                                                              
Des:5 https://dl.yarnpkg.com/debian stable InRelease [17,1 kB]                                                                                                                                                                               
Obj:6 http://ppa.launchpad.net/deadsnakes/ppa/ubuntu bionic InRelease                                                                                                                                                                        
Obj:7 https://baltocdn.com/helm/stable/debian all InRelease                                                                                                                                                                                  
Obj:8 https://repo.skype.com/deb stable InRelease                                                                                                                                                                                            
Obj:9 http://security.ubuntu.com/ubuntu bionic-security InRelease                                                                                                                                                                            
Ign:10 http://packages.microsoft.com/repos/code stable InRelease                                                                                                                                                                             
Obj:12 http://es.archive.ubuntu.com/ubuntu bionic InRelease                                                                                                                                                                                  
Obj:13 http://ppa.launchpad.net/openjdk-r/ppa/ubuntu bionic InRelease                                                                                                                                                
Obj:14 https://packages.microsoft.com/repos/azure-cli bionic InRelease                                                                                                                                                                       
Obj:15 http://packages.microsoft.com/repos/code stable Release                                                                                                                                                                               
Obj:16 http://es.archive.ubuntu.com/ubuntu bionic-updates InRelease                                                                                                                                                                          
Obj:17 https://packages.microsoft.com/repos/ms-teams stable InRelease                                                                                                                                                                       
Obj:18 http://ppa.launchpad.net/zorinos/apps/ubuntu bionic InRelease                                                                                                                    
Obj:19 http://ppa.launchpad.net/zorinos/drivers/ubuntu bionic InRelease                                                                                                             
Obj:20 http://es.archive.ubuntu.com/ubuntu bionic-backports InRelease                                                                                                               
Obj:11 https://packages.cloud.google.com/apt kubernetes-xenial InRelease                                                                                                            
Obj:21 http://ppa.launchpad.net/zorinos/patches/ubuntu bionic InRelease                                                                                   
Obj:22 http://ppa.launchpad.net/zorinos/stable/ubuntu bionic InRelease                                    
Obj:23 https://packages.zorinos.com/stable bionic InRelease                                            
Obj:24 https://packages.zorinos.com/patches bionic InRelease
Obj:25 https://packages.zorinos.com/apps bionic InRelease
Obj:26 https://packages.zorinos.com/drivers bionic InRelease
Descargados 17,1 kB en 4s (4.566 B/s)                  
Leyendo lista de paquetes... Hecho
 2021-02-24T11:41:32.575Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "/usr/bin/apt-get" "dist-upgrade"
Leyendo lista de paquetes... Hecho
Creando árbol de dependencias       
Leyendo la información de estado... Hecho
Calculando la actualización... Hecho
0 actualizados, 0 nuevos se instalarán, 0 para eliminar y 0 no actualizados.
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner   > Step "etc-update"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner   > Step "Brew"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner   > Step "yadm"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner   > Step "nix"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner   > Step "home-manager"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner   > Step "asdf"
 2021-02-24T11:41:33.262Z DEBUG topgrade::utils    > Path "/home/nieto/.config/emacs" doesn't exist
 2021-02-24T11:41:33.262Z DEBUG topgrade::utils    > Path "/home/nieto/.emacs.d" doesn't exist
 2021-02-24T11:41:33.262Z DEBUG topgrade::steps::git > /home/nieto/.doom.d does not exists
 2021-02-24T11:41:33.262Z DEBUG topgrade::steps::git > /home/nieto/.vim does not exists
 2021-02-24T11:41:33.262Z DEBUG topgrade::steps::git > /home/nieto/.config/nvim does not exists
 2021-02-24T11:41:33.262Z DEBUG topgrade::steps::git > /home/nieto/.zshrc does not exists
 2021-02-24T11:41:33.262Z DEBUG topgrade::steps::git > /home/nieto/.tmux does not exists
 2021-02-24T11:41:33.262Z DEBUG topgrade::steps::git > /home/nieto/.config/fish does not exists
 2021-02-24T11:41:33.262Z DEBUG topgrade::steps::git > /home/nieto/.config/openbox does not exists
 2021-02-24T11:41:33.262Z DEBUG topgrade::steps::git > /home/nieto/.config/bspwm does not exists
 2021-02-24T11:41:33.262Z DEBUG topgrade::steps::git > /home/nieto/.config/i3 does not exists
 2021-02-24T11:41:33.262Z DEBUG topgrade::steps::git > /home/nieto/.config/sway does not exists
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "Git repositories"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "zr"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "antibody"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "antigen"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "zplug"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "zinit"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "zim"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "oh-my-zsh"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "fisher"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "bash-it"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "oh-my-fish"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "tmux"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "TLDR"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "pearl"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "SDKMAN!"
 2021-02-24T11:41:33.262Z DEBUG topgrade::utils      > Detected "/bin/bash" as "bash"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "apm"
 2021-02-24T11:41:33.262Z DEBUG topgrade::runner     > Step "vscode"
 2021-02-24T11:41:33.262Z DEBUG topgrade::utils      > Detected "/snap/bin/code" as "code"

―― 12:41:33 - Visual Studio Code ―――――――――――――――――――――――――――――――――――――――――――――――
 2021-02-24T11:41:33.307Z DEBUG topgrade::runner     > Step "vscode" failed: exit code: 1: You are trying to start Visual Studio Code as a super user which isn't recommended. If this was intended, please specify an alternate user data directory using the `--user-data-dir` argument.

 2021-02-24T11:41:33.307Z DEBUG topgrade::terminal   > Desktop notification: vscode failed

Retry? (y)es/(N)o/(s)hell
 2021-02-24T11:41:36.585Z DEBUG topgrade::runner     > Step "vscodium"
 2021-02-24T11:41:36.585Z DEBUG topgrade::runner     > Step "fossil"
 2021-02-24T11:41:36.585Z DEBUG topgrade::runner     > Step "rustup"
 2021-02-24T11:41:36.585Z DEBUG topgrade::runner     > Step ".NET"
 2021-02-24T11:41:36.585Z DEBUG topgrade::runner     > Step "choosenim"
 2021-02-24T11:41:36.585Z DEBUG topgrade::runner     > Step "cargo"
 2021-02-24T11:41:36.585Z DEBUG topgrade::utils      > Detected "/usr/bin/cargo" as "cargo"

―― 12:41:36 - Cargo ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-02-24T11:41:36.585Z DEBUG topgrade::utils      > Detected "/home/nieto/.cargo/bin/cargo-install-update" as "cargo-install-update"
 2021-02-24T11:41:36.585Z DEBUG topgrade::executor   > Running "/home/nieto/.cargo/bin/cargo-install-update" "install-update" "--git" "--all"
    Updating registry 'https://github.com/rust-lang/crates.io-index'

Package       Installed  Latest  Needs update
cargo-update  v5.0.0     v5.0.0  No
topgrade      v6.6.0     v6.6.0  No

No packages need updating.
Package  Installed  Latest  Needs update

No git packages need updating.
Overall updated 0 packages.
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "Flutter"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "Go"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "Emacs"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "opam"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "vcpkg"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "pipx"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "stack"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "tlmgr"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "myrepos"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "jetpack"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "vim"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "Neovim"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "voom"
 2021-02-24T11:41:37.159Z DEBUG topgrade::runner     > Step "npm"
 2021-02-24T11:41:37.159Z DEBUG topgrade::utils      > Detected "/home/nieto/.nvm/versions/node/v14.5.0/bin/npm" as "npm"
 2021-02-24T11:41:37.335Z TRACE topgrade::executor   > Output of "/home/nieto/.nvm/versions/node/v14.5.0/bin/npm" "root" "-g": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/nieto/.nvm/versions/node/v14.5.0/lib/node_modules\n", stderr: "" }
 2021-02-24T11:41:37.335Z DEBUG topgrade::runner     > Step "yarn"
 2021-02-24T11:41:37.335Z DEBUG topgrade::utils      > Detected "/home/nieto/.nvm/versions/node/v14.5.0/bin/yarn" as "yarn"
 2021-02-24T11:41:37.464Z TRACE topgrade::executor   > Output of "/home/nieto/.nvm/versions/node/v14.5.0/bin/yarn" "--version": Output { status: ExitStatus(ExitStatus(0)), stdout: "1.22.10\n", stderr: "" }

―― 12:41:37 - Yarn ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-02-24T11:41:37.465Z DEBUG topgrade::executor   > Running "/home/nieto/.nvm/versions/node/v14.5.0/bin/yarn" "global" "upgrade" "-s"
 2021-02-24T11:41:37.656Z DEBUG topgrade::runner     > Step "deno"
 2021-02-24T11:41:37.656Z DEBUG topgrade::runner     > Step "composer"
 2021-02-24T11:41:37.656Z DEBUG topgrade::runner     > Step "krew"
 2021-02-24T11:41:37.656Z DEBUG topgrade::runner     > Step "gem"
 2021-02-24T11:41:37.656Z DEBUG topgrade::runner     > Step "sheldon"
 2021-02-24T11:41:37.656Z DEBUG topgrade::runner     > Step "rtcl"
 2021-02-24T11:41:37.656Z DEBUG topgrade::runner     > Step "gcloud"
 2021-02-24T11:41:37.656Z DEBUG topgrade::runner     > Step "micro"
 2021-02-24T11:41:37.656Z DEBUG topgrade::runner     > Step "Flatpak"
 2021-02-24T11:41:37.656Z DEBUG topgrade::utils      > Detected "/usr/bin/flatpak" as "flatpak"

―― 12:41:37 - Flatpak User Packages ――――――――――――――――――――――――――――――――――――――――――――
 2021-02-24T11:41:37.656Z DEBUG topgrade::executor   > Running "/usr/bin/flatpak" "update" "--user" "-y"
Looking for updates…
Nothing to do.
 2021-02-24T11:41:37.683Z DEBUG topgrade::executor   > Running "/usr/bin/flatpak" "update" "--system" "-y"
Looking for updates…
Nothing to do.
 2021-02-24T11:41:38.631Z DEBUG topgrade::runner     > Step "snap"
 2021-02-24T11:41:38.631Z DEBUG topgrade::utils      > Detected "/usr/bin/snap" as "snap"

―― 12:41:38 - snap ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-02-24T11:41:38.631Z DEBUG topgrade::executor   > Running "/usr/bin/sudo" "/usr/bin/snap" "refresh"
Todos los snaps están actualizados.
 2021-02-24T11:41:39.307Z DEBUG topgrade::runner     > Step "pihole"
 2021-02-24T11:41:39.307Z DEBUG topgrade::runner     > Step "Firmware upgrades"
 2021-02-24T11:41:39.307Z DEBUG topgrade::runner     > Step "Restarts"

―― 12:41:39 - Summary ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
System update: OK
etc-update: SKIPPED: Cannot find "etc-update" in PATH
Brew: SKIPPED: Cannot find "brew" in PATH
yadm: SKIPPED: Cannot find "yadm" in PATH
nix: SKIPPED: Cannot find "nix" in PATH
home-manager: SKIPPED: Cannot find "home-manager" in PATH
asdf: SKIPPED: Cannot find "asdf" in PATH
Git repositories: SKIPPED: No repositories to pull
zr: SKIPPED: Cannot find "zsh" in PATH
antibody: SKIPPED: Cannot find "zsh" in PATH
antigen: SKIPPED: Cannot find "zsh" in PATH
zplug: SKIPPED: Cannot find "zsh" in PATH
zinit: SKIPPED: Cannot find "zsh" in PATH
zim: SKIPPED: Cannot find "zsh" in PATH
oh-my-zsh: SKIPPED: Cannot find "zsh" in PATH
fisher: SKIPPED: Cannot find "fish" in PATH
bash-it: SKIPPED: Path "/home/nieto/.bash_it" doesn't exist
oh-my-fish: SKIPPED: Cannot find "fish" in PATH
tmux: SKIPPED: Path "/home/nieto/.tmux/plugins/tpm/bin/update_plugins" doesn't exist
TLDR: SKIPPED: Cannot find "tldr" in PATH
pearl: SKIPPED: Cannot find "pearl" in PATH
SDKMAN!: SKIPPED: Path "/home/nieto/.sdkman/bin/sdkman-init.sh" doesn't exist
apm: SKIPPED: Cannot find "apm" in PATH
vscode: FAILED
vscodium: SKIPPED: Cannot find "codium" in PATH
fossil: SKIPPED: Cannot find "fossil" in PATH
rustup: SKIPPED: Cannot find "rustup" in PATH
.NET: SKIPPED: Cannot find "dotnet" in PATH
choosenim: SKIPPED: Cannot find "choosenim" in PATH
cargo: OK
Flutter: SKIPPED: Cannot find "flutter" in PATH
Go: SKIPPED: Cannot find "go" in PATH
Emacs: SKIPPED: Cannot find "emacs" in PATH
opam: SKIPPED: Cannot find "opam" in PATH
vcpkg: SKIPPED: Cannot find "vcpkg" in PATH
pipx: SKIPPED: Cannot find "pipx" in PATH
stack: SKIPPED: Cannot find "stack" in PATH
tlmgr: SKIPPED: tlmgr must be explicity enabled in the configuration to run in Linux
myrepos: SKIPPED: Cannot find "mr" in PATH
jetpack: SKIPPED: Cannot find "jetpack" in PATH
vim: SKIPPED: Cannot find "vim" in PATH
Neovim: SKIPPED: Cannot find "nvim" in PATH
voom: SKIPPED: Cannot find "voom" in PATH
npm: SKIPPED: NPM root at /home/nieto/.nvm/versions/node/v14.5.0/lib/node_modules is owned by 1000 which is not the current user
yarn: OK
deno: SKIPPED: Cannot find "deno" in PATH
composer: SKIPPED: Cannot find "composer" in PATH
krew: SKIPPED: Cannot find "kubectl-krew" in PATH
gem: SKIPPED: Cannot find "gem" in PATH
sheldon: SKIPPED: Cannot find "sheldon" in PATH
rtcl: SKIPPED: Cannot find "rupdate" in PATH
gcloud: SKIPPED: Cannot find "gcloud" in PATH
micro: SKIPPED: Cannot find "micro" in PATH
Flatpak: OK
snap: OK
pihole: SKIPPED: Cannot find "pihole" in PATH
Firmware upgrades: SKIPPED: Cannot find "fwupdmgr" in PATH
Restarts: SKIPPED: Cannot find "needrestart" in PATH
 2021-02-24T11:41:39.308Z DEBUG topgrade::terminal   > Desktop notification: Topgrade finished with errors

</pre></details>
